### PR TITLE
[DOCU-1563] Kong Glossary

### PR DESCRIPTION
**Pull request by Guaris**
### Summary
Adds a glossary.

### Reason
[JIRA](https://konghq.atlassian.net/browse/DOCU-1563)

### CODE REVIEW

**Comment by lena-larionova** [comment]
@Guaris we do have a ticket, it's just not in 3.0: https://konghq.atlassian.net/browse/DOCU-1563

File: src/gateway/understanding-kong/glossary.md [hunk 1]
```diff
@@ -3,4 +3,36 @@ title: Glossary
 
 ---
 
-## PLACEHOLDER Glossary
+| Term      | Definition |
+| ----------- | ----------- |
+|Admin | An Admin is a {{site.base_gateway}} user account capable of accessing the Admin API or Kong Manager. |
+|Authentication| Authentication is the process by which a system validates the identity of a user account. |
+|Authorization| Authorization is the system of defining access to certain resources. In {{site.base_gateway}}, Role-Based Access Control (RBAC) is the main authorization mode. |
+|Beta| A Beta designation in Kong software means the functionality of a feature or release version is of high quality and can be deployed in a non-production environment. |
```
Comment (lena-larionova) [inline src/gateway/understanding-kong/glossary.md:11]: For beta, tech preview/alpha, and GA, might be better to link off to https://deploy-preview-4319--kongdocs.netlify.app/gateway/latest/availability-stages/.

Comment (acgoldsmith) [reply src/gateway/understanding-kong/glossary.md:11]: +1

File: src/gateway/understanding-kong/glossary.md [hunk 2]
```diff
@@ -3,4 +3,36 @@ title: Glossary
...
+|Super Admin| A super admin, or any role with read and write access to the /admins and /rbac endpoints, creates new Roles and customize permissions. A super admin can: invite and disable other admin accounts, assign and revoke roles to admins create new roles with custom permissions, and create new workspaces|
+|Service configuration| The concrete configuration that a runtime group can be configured with in order to expose the service version’s functionality. At the onset, the only type of implementation that we will support will be a Kong proxy.|
+|Spec| A service version can have an optional spec (e.g. OAPI spec for a RESTful service) associated with it.    |
+|Tags| Tags are customer defined labels that let you manage, search for, and filter core entities using the ?tags querystring parameter. Each tag must be composed of one or more alphanumeric characters,` \_\`, `-`, . or `~`. Most core entities can be tagged via their tags attribute, upon creation or edition|
```
Comment (acgoldsmith) [inline src/gateway/understanding-kong/glossary.md:32]:
```suggestion
|Tags| Tags are customer-defined labels that let you manage, search for, and filter core entities using the ?tags querystring parameter. Tags can be added when creating or editing most core entities. Each tag must be composed of one or more alphanumeric characters,` \_\`, `-`, . or `~`. |
```

File: src/gateway/understanding-kong/glossary.md [hunk 3]
```diff
@@ -3,4 +3,37 @@ title: Glossary
...
+|Stable| A Stable release designation in Kong software means the functionality of the version is of high quality, production-ready, and released as general availability (GA). The version has been thoroughly tested, considered reliable to deploy in a production environment, and is fully supported. If updates or bug fixes are required, a patch version or minor release version is issued and fully supported.|
+|Super Admin| A super admin, or any role with read and write access to the /admins and /rbac endpoints, creates new Roles and customize permissions. A super admin can invite and disable other admin accounts, assign and revoke roles to admins, create new roles with custom permissions, and create new workspaces.|
+|Service configuration| The concrete configuration that a runtime group can be configured with in order to expose the service version’s functionality. At the onset, the only type of implementation that we will support will be a Kong proxy.|
+|Spec| A service version can have an optional spec (e.g. OAPI spec for a RESTful service) associated with it.    |
```
Comment (lena-larionova) [inline src/gateway/understanding-kong/glossary.md:31]:
```suggestion
|Spec|  An OpenAPI definition of your service in YAML or JSON format. Can be uploaded to Dev Portal to share with developers in your organization.   |
```

File: src/gateway/understanding-kong/glossary.md [hunk 4]
```diff
@@ -3,4 +3,37 @@ title: Glossary
...
+|Tags| Tags are customer-defined labels that let you manage, search for, and filter core entities using the ?tags querystring parameter. Tags can be added when creating or editing most core entities. Each tag must be composed of one or more alphanumeric characters,` \_\`, `-`, . or `~`. |
```
Comment (lena-larionova) [inline src/gateway/understanding-kong/glossary.md:32]:
```suggestion
|Tags| Tags are customer-defined labels that let you manage, search for, and filter core entities using the `?tags` querystring parameter. Tags can be added when creating or editing most core entities. Each tag must be composed of one or more alphanumeric characters,` \_\`, `-`, . or `~`. |
```

File: src/gateway/understanding-kong/glossary.md [hunk 5]
```diff
@@ -3,4 +3,37 @@ title: Glossary
...
+|Teams| Teams organize developers into working groups.|
+|Tech preview |[availability stages](/gateway/latest/availability-stages) |
```
Comment (lena-larionova) [inline src/gateway/understanding-kong/glossary.md:34]:
```suggestion
|Tech preview | See [availability stages](/gateway/latest/availability-stages). |
```

File: src/gateway/understanding-kong/glossary.md [hunk 6]
```diff
@@ -3,4 +3,37 @@ title: Glossary
...
+|Third-party or community plugin| A custom plugin developed, maintained, and supported by an external developer, not by Kong. Kong does not test these plugins, or update their version compatibility.|
+|Runtime| A Kuma Mesh or a {{site.base_gateway}}, an infrastructure runtime. Previously known as a Kong data plane node     |
```
Comment (lena-larionova) [inline src/gateway/understanding-kong/glossary.md:36]:
```suggestion
```
This is a Konnect term.

File: src/gateway/understanding-kong/glossary.md [hunk 7]
```diff
@@ -3,4 +3,37 @@ title: Glossary
...
+|Service Version| A specific version of a service.|
```
Comment (lena-larionova) [inline src/gateway/understanding-kong/glossary.md:37]:
```suggestion
```
Also a Konnect term.

File: src/gateway/understanding-kong/glossary.md [hunk 8]
```diff
@@ -3,4 +3,37 @@ title: Glossary
...
+|Upstream| An upstream object refers to the API or service managed by Kong, to which client requests are forwarded. An upstream object represents a virtual hostname and can be used to load balance incoming requests over multiple services.|
```
Comment (lena-larionova) [inline src/gateway/understanding-kong/glossary.md:38]:
```suggestion
|Upstream| An upstream object refers to the API or service managed by {{site.base_gateway}}, to which client requests are forwarded. An upstream object represents a virtual hostname and can be used to load balance incoming requests over multiple services.|
```

File: src/gateway/understanding-kong/glossary.md [hunk 9]
```diff
@@ -3,4 +3,37 @@ title: Glossary
...
+|Workspaces| Workspaces enable an organization to segment objects and admins into namespaces. The segmentation allows teams of admins sharing the same Kong cluster to adopt roles for interacting with specific objects.|
```
Comment (lena-larionova) [inline src/gateway/understanding-kong/glossary.md:39]:
```suggestion
|Workspaces| Workspaces enable an organization to segment objects and admins into namespaces. The segmentation allows teams of admins sharing the same {{site.base_gateway}} cluster to adopt roles for interacting with specific objects.|
```

**Review by lena-larionova — CHANGES_REQUESTED** [review]

File: src/gateway/understanding-kong/glossary.md [hunk 10]
```diff
@@ -3,4 +3,37 @@ title: Glossary
...
+|Service| A service, also referred to as a service object, is the upstream APIs and microservices that Kong manages. For example, Services could be a data transformation microservice or a billing API. The main attribute of a Service is its URL, the destination where Kong proxies traffic. The URL can be set as a single string, or by specifying its protocol, host, port, and path.  |
+|Stable| A Stable release designation in Kong software means the functionality of the version is of high quality, production-ready, and released as general availability (GA). The version has been thoroughly tested, considered reliable to deploy in a production environment, and is fully supported. If updates or bug fixes are required, a patch version or minor release version is issued and fully supported.|
+|Super Admin| A super admin, or any role with read and write access to the /admins and /rbac endpoints, creates new Roles and customize permissions. A super admin can invite and disable other admin accounts, assign and revoke roles to admins, create new roles with custom permissions, and create new workspaces.|
```
Comment (lena-larionova) [inline src/gateway/understanding-kong/glossary.md:29]:
```suggestion
|Super admin| A super admin, or any role with read and write access to the /admins and /rbac endpoints, creates new Roles and customize permissions. A super admin can invite and disable other admin accounts, assign and revoke roles to admins, create new roles with custom permissions, and create new workspaces.|
```

**Comment by mheap** [comment]
Failing tests are due to https://github.com/Kong/docs.konghq.com/pull/4325 - rebase and they should pass

**Comment by lena-larionova** [comment]
@Guaris unresolved four missed comments, they're hidden under "Load More" on this page.

**Comment by Guaris** [comment]
> @Guaris unresolved four missed comments, they're hidden under "Load More" on this page.

I think I messed it up during the rebase. added your suggestions.

File: src/gateway/understanding-kong/glossary.md [hunk 11]
```diff
@@ -3,4 +3,32 @@ title: Glossary
...
+|Service| A service, also referred to as a service object, is the upstream APIs and microservices that Kong manages. For example, a service could be a data transformation microservice or a billing API. The main attribute of a service is its URL, the destination where {{site.base_gateway}} proxies traffic. The URL can be set as a single string, or by specifying its protocol, host, port, and path.  |
+|Super admin| A super admin, or any role with read and write access to the /admins and /rbac endpoints, creates new Roles and customize permissions. A super admin can invite and disable other admin accounts, assign and revoke roles to admins, create new roles with custom permissions, and create new workspaces.|
+|Spec|  An OpenAPI definition of your service in YAML or JSON format. Can be uploaded to Dev Portal to share with developers in your organization.   |
+|Tags| Tags are customer-defined labels that let you manage, search for, and filter core entities using the `?tags` querystring parameter. Tags can be added when creating or editing most core entities. Each tag must be composed of one or more alphanumeric characters,` \_\`, `-`, . or `~`. |
```
Comment (lena-larionova) [inline src/gateway/understanding-kong/glossary.md:29]:
```suggestion
|Tags| Tags are customer-defined labels that let you manage, search for, and filter core entities using the `?tags` querystring parameter. Tags can be added when creating or editing most core entities. Each tag must be composed of one or more alphanumeric characters,` \_\`, `-`, `.` or `~`. |
```

**Review by lena-larionova — APPROVED** [review]
One minor edit, otherwise LGTM

**Pull request merged**